build_cell_ontology_dict.R

```diff
@@ -45,6 +45,13 @@ option_list = list(
         type = 'character',
         help = 'Name of the cell ontology terms column in SDRF files (must be identical across all files)'
     ),
+    make_option(
```

Comment:
If you're parameterising this, could you make a it a more generic 'cell type field', than a binary 'use inferred cell type or not'?